Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -1201,6 +1201,14 @@ namespace AzToolsFramework
                 return AZ::Failure(AZStd::string("Failed to duplicate : Couldn't get a valid owning instance for the common root entity of the entities provided."));
             }
 
+            // Block duplicating as overrides since it is not supported.
+            if (!m_prefabFocusHandler.IsOwningPrefabBeingFocused(commonOwningInstance->get().GetContainerEntityId()))
```

Comment:
Is this added here to handle the case of user hitting 'ctrl+d' ?

Reply:
Yes. This can be helpful in automated tests as well.

'ctrl+d' is handled by SandboxIntegrationManager but I did not make changes to that class.